lib/Bloc/max_bloc.dart

```diff
@@ -4,12 +4,12 @@ import 'package:exerlog/Models/exercise.dart';
 import 'package:exerlog/Models/maxes.dart';
 import 'package:exerlog/Models/sets.dart';
 
-import '../main.dart';
+import '../src/core/base/shared_preference/shared_preference_b.dart';
 
 Future<List<Max>> getSpecificMax(String exercise, double reps) async {
   final ref = await FirebaseFirestore.instance
       .collection('users')
-      .doc(userID)
+      .doc(SharedPref.getStringAsync(USER_UID))
```

Comment:
Since it is used multiple times here, we can declare a top-level `String` `variable` and store the value there and use it everywhere else in this file.